languages/elixir/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,78 @@
+In Elixir, a tuple is a data structure which organizes data in contiguous memory. It behaves as like static data structures, holding a fixed number of items, but doesn't have explicit names for each element. Tuples are often used in Elixir for memory read-intensive operations, since read-access of an element is a constant-time operation. They are not usually used when elements may need to be added/removed dynamically because rather than modifying the existing tuple, a new tuple is created which requires memory to be allocated upfront.
```

Comment:
Nice introduction! Would it be useful to mention that tuples members can be of different types?